test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -400,6 +438,9 @@ private class TestController
             [ValueProviderMetadata]
             public string ValueBinderMarkedProperty { get; set; }
 
+            [CustomBindingSource]
```

Comment:
Yeah, my slight preference is to just leave stuff like this out if it's not connected to anything. Gives the impression that it's magic :tophat: